pages/cpm.vue

```diff
@@ -1,24 +1,34 @@
 <template>
   <div class="flex place-content-center">
-    <div id="mapid" style="height: 400px; width: 450px" />
+    <div id="mapid" style="height: 600px; width:500px" />
   </div>
 </template>
 
 <script>
 import 'leaflet/dist/leaflet.css'
 import L from 'leaflet'
-import 'proj4leaflet'
+// import 'proj4leaflet'
 
 export default {
   data () {
     return {
-      map: {}
+      map: {},
+      info: {},
+      // groups and experiments over different regions
+      groups: {
+        NW: 'CNRM, KNMI, ETH, UKMO',
+        SW: 'CMCC, IPSL, ETH, UKMO',
+        SE: 'ICTP, ETH, UKMO',
+        C: 'GERICS, ETH, UKMO',
+        CE: 'SMHI, ICTP, ETH, UKMO',
+        N: 'DMI/SMHI, GERICS'
+      }
```

Comment:
It might be nice if this information is combined with the domain coordinates info. At some point we might want to move that into a separate (geojson?) file.

Reply:
> 

Good point! I will include the coordinates info. And yes, I think it is better later on we move it to a geojson (or maybe simply a yaml file) to make it cleaner.